packages/@okta/vuepress-site/docs/guides/byo-saml/overview/index.md

```diff
@@ -14,13 +14,13 @@ To use your own SAML certificate, update the key credential for the affected app
 
 The general procedure is the same for Outbound and Inbound SAML application; however, some of the api calls are different, as described in the steps below. The general procedure contains the following seven steps:
 
-  1. [List your apps](#step-1-list-your-apps)
-  2. [Generate a certificate signing request (CSR)](#step-2-generate-a-csr)
-  3. [Sign the CSR](#step-3-sign-the-csr)
-  4. [Publish the CSR](#step-4-publish-the-csr)
-  5. [Update the key credential for the app to specify the new certificate](#step-5-update-the-key-credential)
-  6. [Clone the certificate (optional)](#step-6-clone-the-certificate-optional)
-  7. [Upload the new certificate to the ISV](#step-7-upload-the-new-certificate-to-the-isv)
+  1. [List your apps](../list-your-apps)
```

Comment:
Have you confirmed these links work as intended (Links in guides have some quirks).    See https://github.com/okta/okta-developer-docs#linking-between-sections-and-guides for how these links should build.

Reply:
On local, the URLs worked successfully. Nevertheless, I've modified them with the full path and trailing slash in order to follow the standard.